### ISSUE TITLE
[Bug]: The insert execution with high concurrency is not stable 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#3781]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.5.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

During the high concurrency execution of insert, TPS will suddenly drop sharply until it drops to 0, and  During this period, the update operation of the table will be blocked. A moment later,it can recovery

**Comment by XuPeng-SH** [comment]
Performance issue

**Comment by XuPeng-SH** [comment]
0.7 scope

**Comment by XuPeng-SH** [comment]
0.7

**Comment by XuPeng-SH** [comment]
0.7 and CN scope

**Comment by XuPeng-SH** [comment]
It was tested based on 0.5.
After checking with @aressu1985, assign it back

**Comment by aressu1985** [comment]
this can be closed, and will re test in 0.0.7
